src/machine.rs

```diff
@@ -983,6 +990,20 @@ impl<'mir, 'tcx> Machine<'mir, 'tcx> for Evaluator<'mir, 'tcx> {
             // Cannot overflow, since it is strictly less than `report_progress`.
             ecx.machine.since_progress_report += 1;
         }
+
+        // Search for SbTags to find all live pointers, then remove all other tags from borrow
+        // stacks.
+        // When debug assertions are enabled, run the GC as often as possible so that any cases
+        // where it mistakenly removes an important tag become visible.
+        if cfg!(debug_assertions)
```

Comment:
This increases CI times by a fair bit, 15 minutes to 21 minutes it looks like. But without this, I'm not really sure how else to find problems with the GC in tests.

Reply:
maybe only turn it on for a single test suite (like x86_64-linux on the x86_64-linux host) by setting the gc miri flag for it via env vars?

Reply:
this hasn't happened yet afaict